### ISSUE TITLE
OpenBSD:  LuaSnip & telescope-fzf-native.nvim fail to build and install

### ISSUE THREAD

**Issue by holsta** [nvim-lua/kickstart.nvim#706]
## Describe the bug
LuaSnip and telescope-fzf-native fail to build when making use of kickstart.nvim on OpenBSD.

## To Reproduce
```
$ uname -a
OpenBSD foo.example.com 7.5 GENERIC.MP#46 amd64
$ git clone https://github.com/nvim-lua/kickstart.nvim ~/.config/nvim
Cloning into '/home/neovim/.config/nvim'...
remote: Enumerating objects: 833, done.
remote: Counting objects: 100% (21/21), done.
remote: Compressing objects: 100% (18/18), done.
Receiving objects: 100% (833/833), 387.38 KiB | 2.02 MiB/s, done.
remote: Total 833 (delta 8), reused 10 (delta 3), pack-reused 812
Resolving deltas: 100% (414/414), done.

$ nvim
[installation continues inside neovim]

             Failed (2)
               ● LuaSnip 27.3ms ✔ build     ■■ build failed
                   *** Parse error in /home/neovim/.local/share/nvim/lazy/LuaSnip: Missing dependency
                   operator (Makefile:37)
                   *** Parse error: Need an operator in 'endif' (Makefile:42)
               ● telescope-fzf-native.nvim 0.17ms ✔ build     ■■ build failed
                   *** Parse error in /home/neovim/.local/share/nvim/lazy/telescope-fzf-native.nvim:
                   Missing dependency operator (Makefile:3)
                   *** Parse error: Missing dependency operator (Makefile:6)
                   *** Parse error: Need an operator in 'else' (Makefile:10)
                   *** Parse error: Need an operator in 'endif' (Makefile:13)
                   *** Parse error: Need an operator in 'else' (Makefile:14)
                   *** Parse error: Need an operator in 'endif' (Makefile:18)

             Installed (21)
               ● cmp-nvim-lsp 0.17ms 󰢱 cmp_nvim_lsp  nvim-lspconfig     ■ already up to date
               ○ cmp-path  nvim-cmp      ■ already up to date
               ○ cmp_luasnip  nvim-cmp      ■ already up to date
               ● Comment.nvim 6.36ms  start     ■ already up to date
               ● conform.nvim 2.71ms  start     ■ already up to date
               ● fidget.nvim 57.62ms  nvim-lspconfig     ■ already up to date
               ● gitsigns.nvim 4.23ms  start     ■ already up to date
               ● mason-lspconfig.nvim 0.91ms  nvim-lspconfig     ■ already up to date
               ● mason-tool-installer.nvim 23.1ms  nvim-lspconfig     ■ already up to date
               ● mason.nvim 1ms  nvim-lspconfig     ■ already up to date
[quit out of neovim]

$ nvim
stylua: failed to install
lua-language-server: failed to install
Press ENTER or type command to continue
```

Output from :checkhealth includes:

WARNING stylua unavailable: Command not found

## Desktop
- OS: OpenBSD
- Terminal: xterm

## Neovim Version
```
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1692716794

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"
```

**Comment by schieni** [comment]
Regarding stylua: I also had this error. The unzip command was missing.

**Comment by feoh** [comment]
Believe it or not, I actually encountered the fzf issue when I tried running on NetBSD because the SDF uses that.

You're going to need to do some poking around yourself, because I'm unsure if we have anyone who contributes regularly who even has an OpenBSD system to look at for this.

Here's the snippet I had to put in my .zshrc to make it work:

```
if [[ `uname -s` == "NetBSD" ]]; then
        export FZF_BASE="/usr/pkg/share/fzf"
fi
source $ZSH/oh-my-zsh.sh
```

LuaSnip has some requirements documented [here](https://github.com/L3MON4D3/LuaSnip). Be sure you have those installed and in your PATH.
